data/remotes.d/fwupd.conf

```diff
@@ -4,4 +4,4 @@
 Enabled=true
 Type=local
 Keyring=none
-File=@datadir@/fwupd/remotes.d/fwupd/metadata.xml
+MetadataURI=file://@datadir@/fwupd/remotes.d/fwupd/metadata.xml
```

Comment:
Agree, but after trying to test all the ways people could get this wrong I didn't fancy supporting `File=http://` or `Type=download` with `MetadataURI=file://` -- saying "this is the one way to do it" makes the testing matrix much smaller.